pkgs/applications/altcoins/wownero.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchgit, cmake, pkgconfig, git
+, boost, miniupnpc, openssl, unbound, cppzmq
+, zeromq, pcsclite, readline, libsodium
+, CoreData, IOKit, PCSC
+}:
+
+assert stdenv.isDarwin -> IOKit != null;
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "wownero-${version}";
+
+  version = "0.3.1.3";
+  src = fetchgit {
+    url = "https://github.com/wownero/wownero";
+    fetchSubmodules = true;
+    rev    = "v${version}";
+    sha256 = "1n3njbgyw7iklaybh8v6hlaj6qjmqich4g1gbyippxh8vncwasay";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig git ];
+
+  buildInputs = [
+    boost miniupnpc openssl unbound
+    cppzmq zeromq pcsclite readline libsodium
+  ] ++ optionals stdenv.isDarwin [ IOKit CoreData PCSC ];
+
+  cmakeFlags = [
+    "-DReadline_ROOT_DIR=${readline.dev}"
+    "-DMANUAL_SUBMODULES=ON"
+  ] ++ optional stdenv.isDarwin "-DBoost_USE_MULTITHREADED=OFF";
+
+  hardeningDisable = [ "fortify" ];
+
+  meta = {
+    description = "Many coins, such privacy. Wow!";
+    longDescription = ''
+      Wownero is a fork of the cryptocurrency Monero with primary alterations.
+
+      Wownero’s emission is capped and supply is finite.
+
+      Wownero is a fairly launched coin with no premine.
+
+      It’s not a fork of another blockchain. With its own genesis block there is no degradation of privacy caused by ring signatures using different participants for the same transaction outputs.
+
+      Unlike opposing forks.
+    '';
+    homepage    = https://wownero.org/;
```

Comment:
They don't respond to https, http://wownero.org/ works though.

Reply:
Thanks, reported in the IRC, but it should be fixed soon.

Reply:
url updated to use plain http.